doc/source/user_guide/indexing.rst

```diff
@@ -1362,7 +1362,8 @@ Of course, expressions can be arbitrarily complex too:
    shorter = df.query('a < b < c and (not bools) or bools > 2')
 
    # equivalent in pure Python
-   longer = df[(df.a < df.b) & (df.b < df.c) & (~df.bools) | (df.bools > 2)]
+   longer = df[(df['a'] < df['b']) & (df['b'] < df['c'])
```

Comment:
use a multi-line format to make this readable

Reply:
Do you mean using `/`? `flake8-rst` complained about that being redundant.

Reply:
no, we *never* use ``/``, add parens if needed, put each condition on its own line (this is how code would be formatted), e.g something like this

```df[((df......) 
           & (df....)
           & (.......))]```

Reply:
well lined up :->

Reply:
Got it, thanks for explaining :-)